python/src/opendp/domains.py

```diff
@@ -207,6 +208,44 @@ def domain_type(
     return output
 
 
+def expr_domain(
+    lazyframe_domain: Domain,
+    context: Optional[str] = None,
+    grouping_columns: Optional[Any] = None
+) -> Domain:
+    r"""Construct an ExprDomain from a LazyFrameDomain.
+
+    Must pass either `context` or `grouping_columns`.
```

Comment:
This is generated code, I think? If it were manual, I'd expect there to be a check if both `context` and `grouping_columns` have been provided. I guess that would come from the Rust code?

Reply:
Yep, those checks happen in the Rust code.